independent-publisher-connectors/Proposal - Github as Datasource/readme.md

```diff
@@ -0,0 +1,38 @@
+## Github as a datassource
+
```

Comment:
Can you please modify the readme.md file as per the template below?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md